csv-generator/deployment/src/main/java/io/quarkiverse/operatorsdk/csv/deployment/ManifestsProcessor.java

```diff
@@ -177,6 +216,42 @@ private CSVMetadataHolder createMetadataHolder(AnnotationInstance csvMetadata, C
             maintainers = mh.maintainers;
         }
 
+        final var installModesField = csvMetadata.value("installModes");
+        CSVMetadataHolder.InstallMode[] installModes = null;
+        if (installModesField != null) {
+            final var installModesAnn = installModesField.asNestedArray();
+            installModes = new CSVMetadataHolder.InstallMode[installModesAnn.length];
+            for (int i = 0; i < installModesAnn.length; i++) {
+                installModes[i] = new CSVMetadataHolder.InstallMode(
+                        ConfigurationUtils.annotationValueOrDefault(installModesAnn[i], "type",
+                                AnnotationValue::asString, () -> null),
+                        ConfigurationUtils.annotationValueOrDefault(installModesAnn[i], "supported",
+                                AnnotationValue::asBoolean, () -> true));
+            }
+        } else {
+            installModes = mh.installModes;
+        }
+
+        final var permissionsField = csvMetadata.value("permissionRules");
+        CSVMetadataHolder.PermissionRule[] permissionRules = null;
+        if (permissionsField != null) {
+            final var permissionsAnn = permissionsField.asNestedArray();
+            permissionRules = new CSVMetadataHolder.PermissionRule[permissionsAnn.length];
+            for (int i = 0; i < permissionsAnn.length; i++) {
+                permissionRules[i] = new CSVMetadataHolder.PermissionRule(
+                        ConfigurationUtils.annotationValueOrDefault(permissionsAnn[i], "apiGroups",
+                                AnnotationValue::asStringArray, () -> null),
+                        ConfigurationUtils.annotationValueOrDefault(permissionsAnn[i], "resources",
+                                AnnotationValue::asStringArray, () -> null),
+                        ConfigurationUtils.annotationValueOrDefault(permissionsAnn[i], "verbs",
+                                AnnotationValue::asStringArray, () -> null),
```

Comment:
It doesn't appear that if no verbs are provided, then all verbs are used as the default…

Reply:
ok